static/js/utils/place_utils.ts

```diff
@@ -165,15 +167,16 @@ export function getPlaceNames(
   if (!dcids.length) {
     return Promise.resolve({});
   }
-  let url = "/api/place/name?";
-  const urls = [];
-  for (const place of dcids) {
-    urls.push(`dcid=${place}`);
-  }
-  url += urls.join("&");
-  return axios.get(url).then((resp) => {
-    return resp.data;
-  });
+  return axios
+    .get("/api/place/name", {
+      params: {
+        dcids: dcids,
+      },
+      paramsSerializer: stringifyFn,
+    })
+    .then((resp) => {
+      return resp.data;
+    });
```

Comment:
Should we add an error handler here, that returns just the dcid's? Could be a TODO as well.

Reply:
Yes, we should. I will file a bug to fix all the axios errors handling.

Reply:
sounds good - thanks!